.ci/scripts/test.sh

```diff
@@ -213,7 +213,7 @@ elif [[ -n "${TAV_MODULE}" ]]; then
     memcached)
       DOCKER_COMPOSE_FILE=docker-compose-memcached.yml
       ;;
-    aws-sdk|@aws-sdk/client-s3)
+    aws-sdk|@aws-sdk/client-s3|@aws-sdk/client-dynamodb)
```

Comment:
Note for reviewer: I thought about using a wildcard to match all clients but not sure if it's better to be explicit. Thoughts?